src/components/View.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types, react/prefer-stateless-function */
```

Comment:
- [ ] Please, consider removing all linter disables - let the linter makes its direct job, aka help you find bugs in the early stages of development 👍